loleaflet/src/control/Control.UserList.js

```diff
@@ -81,6 +81,12 @@ L.Control.UserList = L.Control.extend({
 		var content = L.DomUtil.create('tr', 'useritem');
 		content.id = 'user-' + viewId;
 		$(document).on('click', '#' + content.id, this.onUseritemClicked.bind(this));
+		var ownerClass = userName == 'You' ? 'onwer' : 'guess';
+
+		$('#listActiveUser').append('<div class="user-item-wrapper" id="user-wrapper-' + viewId + '"><p class="user-item" id="user-' + viewId 
+		+ '" style="background-image:url(\'' + extraInfo.avatar + '\');border-color:' + color + '"></p><span> '+ userName +'</span></div>');
+
+		$('.userInfo').append('<p class="user-top ' + ownerClass + '" id="user-top-' + viewId + '" style="background-image:url(\'' + extraInfo.avatar + '\');border-color:' + color + ';"></p>');
```

Comment:
Would be great if we could move the rendering of the header avatars to a separate method, ideally also just making use of plain javascript for the dom manipulation instead of jquery, so this doesn't have the external dependency on the library.

Reply:
A few other points to consider:
- The avatar might not be available https://github.com/CollaboraOnline/online/blob/3ab824cdfde71c79d627bbe7c69f47610fed6019/loleaflet/src/control/Control.UserList.js#L94
- For uniquely identifying the element you could use a data-view-id="" attribute